src/main/java/com/movie_collection/dal/dao/CategoryDAO.java

```diff
@@ -0,0 +1,49 @@
+package com.movie_collection.dal.dao;
+
+import com.movie_collection.be.Category;
+import com.movie_collection.dal.ConnectionManager;
+import com.movie_collection.dal.interfaces.ICategoryDAO;
+import javafx.beans.property.SimpleStringProperty;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CategoryDAO implements ICategoryDAO {
+
+    private static final ConnectionManager cm = new ConnectionManager();
+
+    public List<Category> getAllCategories() throws SQLException {
+        ArrayList<Category> allCategories = new ArrayList<>();
+        try (Connection connection = cm.getConnection()) {
+            String sql = "SELECT * FROM ALL_PLAYLISTS order By id asc";
```

Comment:
why is the table called ALL_PLAYLISTS ? since its all CAPS and also incorrect naming ....